### PR TITLE
added-border-animation-file

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

**Review by OSAMA478 — APPROVED** [review]
Great animation.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+body {
+  min-height: 100vh;
+  background: rgb(17, 17, 17);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.frem {
```
Comment (Tauqeer-Ahmed-99) [inline style.css]: spelling mistake frame.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  width: 16rem;
+  height: 16rem;
+  overflow: hidden;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.frem .rotat {
```
Comment (Tauqeer-Ahmed-99) [inline style.css]: use meaningful and proper class and Id names

File: style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+.frem {
+  width: 16rem;
+  height: 16rem;
+  overflow: hidden;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
```
Comment (Tauqeer-Ahmed-99) [inline style.css:24]:
```suggestion
  width: 16rem;
  height: 16rem;

  overflow: hidden;

  display: flex;
  justify-content: center;
  align-items: center;
  
  position: relative;
```

Comment (Tauqeer-Ahmed-99) [inline style.css:24]: if you are grouping properties, then group only similar properties together

File: style.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+}
+.frem .rotat {
+  width: 30rem;
+  height: 15rem;
+  background: #00ff88;
+  position: absolute;
+  animation: rotats 3s infinite ease-in-out;
```
Comment (Tauqeer-Ahmed-99) [inline style.css]: width: 30rem;
  height: 15rem;

  background: #00ff88;

  position: absolute;

  animation: rotats 3s infinite ease-in-out;

Comment (Tauqeer-Ahmed-99) [inline style.css]: typo in line  29 rotate.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  z-index: 2;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.frem .cant .area img {
+  filter: invert(1);
+  width: 6rem;
+  border-radius: 50%;
```
Comment (Tauqeer-Ahmed-99) [inline style.css:60]: group similar properties only

File: style.css (hunk 1 above)
Comment (murtazshaikh) [reply style.css]: Changed

File: style.css (hunk 2 above)
Comment (murtazshaikh) [reply style.css]: Done

File: style.css (hunk 3 above)
Comment (murtazshaikh) [reply style.css:24]: Thankyou @Tauqeer-Ahmed-99 for showing how to group similar properties, now I got the clear Idea.

Comment (murtazshaikh) [reply style.css:24]: Done

File: style.css (hunk 4 above)
Comment (murtazshaikh) [reply style.css]: Done

Comment (murtazshaikh) [reply style.css]: Changed

File: style.css (hunk 5 above)
Comment (murtazshaikh) [reply style.css:60]: okk

**Comment by murtazshaikh** [comment]
> Great animation.

thanks @OSAMA478

**Review by syedabdullahali — APPROVED** [review]

**Review by OSAMA478 — APPROVED** [review]
nice animation.

**Pull request merged**